ask.py

```diff
@@ -0,0 +1,31 @@
+import os
+import openai
+from dotenv import load_dotenv
+
+
+load_dotenv()
+openai.api_key = os.getenv("OPENAI_API_KEY")
+
+def main():
+
+
+
+    messages=[
+                {"role": "system", "content": "You are a leagal assistant"},
+            ]
+
+    while True:
+        message = input("User : ")
+        if message:
+            messages.append(
+                {"role": "user", "content": message},
+            )
+            chat = openai.ChatCompletion.create(
+                model="gpt-3.5-turbo", messages=messages
+            )
+        
+        reply = chat.choices[0].message.content
+        print(f"ChatGPT: {reply}")
+        messages.append({"role": "assistant", "content": reply})
+
+main()        
```

Comment:
add an `if` statement to run the function only if you run the file (`if __name__ ...`)